# name: PyExc_InterruptedErr 
PyExc_InterruptedError